src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -7,19 +7,21 @@
 <div class="f8-wi__list-entry">
 	<!-- info area -->
   <div class="list-view-pf-main-info">
-    <div class="list-view-pf-left type f8-wi__list-witype">
-      <span class="pull-left">
+    <div class="list-view-pf-left type pull-left f8-wi__list-witype">
+      <span class="pull-left margin-0">
```

Comment:
Confused by this - why have the child element as `pull-left` when the parent element is already set to `pull-left`? Unless you are setting things to align to the right, they should default to left alignment.